packages/deb/hazelcast/DEBIAN/preinst

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+java -version || (echo "Java must be installed" && exit 1)
```

Comment:
Do we need it if we define jdk as a dependency?

Reply:
no, I've removed it